dandi/support/bids/validator.py

```diff
@@ -689,12 +701,42 @@ def log_errors(validation_result):
         lgr.warning("The `%s` file was not matched by any regex schema entry.", i)
 
 
+def _get_directory_suffixes(my_schema):
+    """Query schema for suffixes which identify directory entities.
+
+    Parameters
+    ----------
+    my_schema : dict
+        Nested directory as produced by `schemacode.schema.load_schema()`.
+
+    Returns
+    -------
+    list of str
+        Directory pseudofile suffixes excluding trailing slashes.
+
+    Notes
+    -----
+    * Yes this seems super-awkward to do explicitly, after all, the trailing slash is
+        already in so it should automagically work, but no:
+        - Subdirectory names need to be dynamically excluded from validation input.
+        - Backslash directory delimiters are still in use, which is regrettable.
+    """
+    pseudofile_suffixes = []
+    for i in my_schema["objects"]["extensions"].values():
+        i_value = i["value"]
+        if i_value.endswith("/"):
+            if i_value != "/":
+                pseudofile_suffixes.append(i_value[:-1])
+    return pseudofile_suffixes
+
+
 def validate_bids(
     bids_paths,
-    schema_reference_root="{module_path}/support/bids/schemadata/",
+    schema_reference_root="/usr/share/bids-schema/",
```

Comment:
Under what conditions is this default used, and why would we expect anything to be in this directory?